packages/data-stores/src/queries/use-is-domain-code-valid.ts

```diff
@@ -0,0 +1,25 @@
+import { useQuery } from '@tanstack/react-query';
```

Comment:
👋🏻 @alshakero the `@automattic/data-stores` package doesn't seem to be the right fit for custom query hooks and mutations based on `@tanstack/query` which is why we attempt to move a few of them in https://github.com/Automattic/wp-calypso/pull/78682 to where they are used. I'd suggest following up on this PR and move those to where they are used. Please feel free to ping @Automattic/team-calypso-frameworks for a review or let us know if we can help with anything.

Reply:
Moved into Calypso!